uefi/src/lib.rs

```diff
@@ -115,6 +135,74 @@
 //! Contributions in the form of a PR are also highly welcome. Check our
 //! [contributing guide][contributing] for details.
 //!
+//! # Comparison to other Projects in the Ecosystem
+//!
+//! ## Rust `std` implementation
+//!
+//! There is an ongoing effort for a [`std` implementation][rustc-uefi-std] of
+//! the Rust standard library, which allows you to write UEFI programs that look
+//! very similar to normal Rust programs running on top of an OS.
+//!
+//! It is still under development. You can track the progress in the
+//! corresponding [tracking issue][uefi-std-tr-issue].
+//!
+//! Using the `std` implementation simplifies the overall process of producing
+//! the binary. For example, our [`#[entry]`][entry-macro] macro won't be
+//! required any longer. As the `std` implementation evolves over time, you'll
+//! need fewer and fewer abstractions of this crate. For everything not covered
+//! by the `std` implementation, you can obtain relevant structures to work with
+//! our crate via:
+//! - `std::os::uefi::env::boot_services()`
+//! - `std::os::uefi::env::get_system_handle()`
+//! - `std::os::uefi::env::get_system_table()`
+//!
+//! ### Example
+//!
+//! A minimal Rust app utilizing the uefi `std` implementation and leveraging
+//! functionality from `uefi` might look like as follows:
+//!
+//! ```ignore
+//! #![feature(uefi_std)]
+//! #![feature(uefi_std)]
+//!
+//! use std::os::uefi as uefi_std;
+//! use uefi::{Handle, Status};
+//! use uefi::runtime::ResetType;
+//!
+//! /// Performs the necessary setup code for the `uefi` crate.
+//! fn setup_uefi_crate() {
+//!     let st = uefi_std::env::system_table();
+//!     let ih = uefi_std::env::image_handle();
+//!
+//!     // Mandatory setup code for `uefi` crate.
+//!     unsafe {
+//!         uefi::table::set_system_table(st.as_ptr().cast());
+//!
+//!         let ih = Handle::from_ptr(ih.as_ptr().cast()).unwrap();
+//!         uefi::boot::set_image_handle(ih);
+//!     }
+//! }
+//!
+//! fn main() {
+//!     println!("Hello World from uefi_std");
+//!     setup_uefi_crate();
+//!     println!("UEFI-Version is {}", uefi::system::uefi_revision());
+//!     uefi::runtime::reset(ResetType::SHUTDOWN, Status::SUCCESS, None);
+//! }
+//! ```
+//!
+//! ## `r-efi`
+//!
+//! [`r-efi`] provides Raw UEFI bindings without high-level convenience similar
+//! to our `uefi-raw` crate, which is part of this  project, but more
+//! feature-complete. It targets a lower-level than our `uefi` crate does.
+//!
+//! # MSRV
```

Comment:
The MSRV section was already added to `lib.rs` in a previous PR, so this can be dropped.

Reply:
Sorry, artifact from a big rebase - I missed that.

Reply:
The duplicate MSRV section is still present

Reply:
What?! How? I definitely removed that locally.

Sorry.. I really have no clue how this happened.